### ISSUE TITLE
Option to Include Alert Details in Slack Messages

### ISSUE THREAD

**Issue by mastercactapus** [target/goalert#3314]
**What problem would you like to solve? Please describe:**
Currently, messages sent via Slack include only the summary of the alert. The details are frequently useful when responding to incidents and having them directly in the Slack message would make that information more quickly accessible.

**Describe the solution you'd like:**
A new feature that gives users the option to include the details of alerts in Slack messages. Given the concern about potential extra noise in Slack channels, these additional details could be collapsed by default, and possibly remain collapsed once the alert is acknowledged or closed.

**Describe alternatives you've considered:**
Email and Webhook destinations currently include the details field already. However, enhancing this feature for Slack specifically is desirable to improve incident management flow.

**Additional context:**
The code for Slack notifications is under `notification/slack/channel.go` in the `Send` method.

**Comment by mastercactapus** [comment]
Not sure if there's a way to add collapsible content, but maybe when sending a first message, we can put the details as a thread reply to keep things from getting noisy in a channel.

**Comment by yes-chris** [comment]
Expanded Slack details would be a huge help to people that receive alerts during off hours. The summary that is currently sent is very limited and can almost be a hinderance at times.

**Comment by audunmg** [comment]
I also agree, and I hope you can consider and comment on @Chin-Lee-lol's PR 👍 

It's a nice step in the right direction IMHO
